panelinhadees/client
# Implement login and logout

## Related Issue
Resolves #13 and Closes #19 

## What is the purpose of this pull request?
Add login and logout functionality using a button in the initial page and make a query retrieving the user data. Save the user token in Apollo Cache.

**What I did:**

- Created `LoginButton` and `LogoutButton`
- Removed lint check on dev mode
- Added the `localhost:5000/graphql` as our server
- Made a query to retrieve user data
- Created a middleware to check if the user is authenticated in a route.
- Add tests with jest

## What problem does it solve?
Login and query using the token.

## How Has This Been Tested?
1. Go to the home page
2. Login
3. See if the button changed to `logout`
4. go to the page `/settings` and see if appears the data.

Check the IsAuthenticated middleware:

1. Try to access `/settings` without being logged, you should see an error.

## Screenshots or example usage:
https://drive.google.com/file/d/1R-nXSAZEAu69JLrF2SADgU3knC5eIzQ3/view

## Types of changes
